src/swig/CMakeLists.txt

```diff
@@ -33,6 +33,9 @@ if(BUILD_SWIG_PYTHON OR BUILD_SWIG_OCTAVE OR BUILD_SWIG_MATLAB)
   if (HAVE_JSON)
     SET(CMAKE_SWIG_FLAGS ${CMAKE_SWIG_FLAGS} -DHAVE_JSON)
   endif()
+  if (HAVE_ITK)
```

Comment:
Let's stick to `ITK_FOUND`

Reply:
happy to switch - should I do it in all places, also where HAVE_ITK was used previously?

Reply:
as above